frontend/src/components/Profile/BasicTabs.tsx

```diff
@@ -13,7 +16,6 @@ const defaultProps = {
 
 function TabPanel(props: TabPanelProps & typeof defaultProps) {
   const { children, value, index } = props;
```

Comment:
do it like this

![image](https://user-images.githubusercontent.com/53787083/169655576-086e2d16-b0e5-482d-b36f-2bd3a078b5a0.png)